darwin/torch/transforms.py

```diff
@@ -364,8 +378,15 @@ def _post_process(self, albumentation_output: dict, annotation: dict) -> tuple:
         bboxes = albumentation_output.get("bboxes")
         if bboxes is not None:
             output_annotation["boxes"] = torch.tensor(bboxes)
+
             if "area" in annotation and "masks" not in albumentation_output:
```

Comment:
just wondering, why is it necessary to check for "masks" not being in the dict here?

Reply:
If masks are present, area calculated from the masks are more accurate then an area from bounding boxes.